railties/lib/rails/generators/rails/app/templates/.devcontainer/docker-compose.yml.tt

```diff
@@ -0,0 +1,78 @@
+services:
```

Comment:
What about naming this file `compose.yaml` (it has been [the default name](https://docs.docker.com/compose/compose-application-model/#the-compose-file) for Compose configuration for a while).

Reply:
And, we need data `volumes:` for the database?!